components/Navbar.vue

```diff
@@ -1,43 +1,50 @@
 <template>
-  <header class="navbar">
-    <nav>
-      <div class="box">
-        <div class="logo">
-          <a href="https://realdevsquad.com/">
-            <img
-              src="~/assets/Real-Dev-Squad@1x.png"
-              alt="Real Dev Squad logo"
-            />
-          </a>
-        </div>
-        <div class="hamburger-menu" @click="toggleClicked(isClicked)">
-          <span></span>
-          <span></span>
-          <span></span>
-        </div>
-      </div>
-      <ul :class="[isClicked ? 'show-menu' : 'hide-menu']">
-        <li>
-          <a href="https://welcome.realdevsquad.com/">Welcome</a>
-        </li>
-        <li>
-          <a href="https://www.realdevsquad.com/events.html">Events</a>
-        </li>
-        <li>
-          <a href="https://members.realdevsquad.com/">Members</a>
-        </li>
-        <li>
-          <a href="https://crypto.realdevsquad.com/">Crypto</a>
-        </li>
-        <li>
-          <a href="https://status.realdevsquad.com/">Status</a>
-        </li>
-        <li>
-          <a class="active" href="https://goals.realdevsquad.com/">Goals</a>
-        </li>
-      </ul>
-    </nav>
-  </header>
+  <nav class="navbar">
+    <button class="hamburger" @click="toggleClicked(isClicked)">
+      <div class="hamburger__bar" />
+      <div class="hamburger__bar" />
+      <div class="hamburger__bar" />
+    </button>
+
+    <ul class="redirection">
+      <li>
+        <a href="https://realdevsquad.com/" class="redirection__option--logo">
+          <img
+            src="~/assets/Real-Dev-Squad@1x.png"
+            alt="Real Dev Squad logo"
+            width="50px"
+            height="50px"
+          />
+        </a>
+      </li>
+      <li v-for="item in LINKS" :key="item.name">
+        <a :href="item.link" class="redirection__option">{{ item.name }}</a>
+      </li>
+    </ul>
+
+    <ul class="dropdown" :class="{ 'dropdown--open': isClicked }">
+      <li>
+        <a href="https://realdevsquad.com/" class="dropdown__option">Home</a>
+      </li>
+      <li v-for="item in LINKS" :key="item.name">
+        <a :href="item.link" class="dropdown__option">{{ item.name }}</a>
+      </li>
+    </ul>
```

Comment:
Do let me know if there is a way to merge this two into one.

Reply:
Yes we can merge this by using two separate elements for `Home` as we are using here and set the `display: none` and `display: block` properties as per the screen sizes using the media query
And for the rest of the links we can use simple CSS for the appearance with media query

Reply:
You can refer to this PR or else we can connect on Discord
https://github.com/Real-Dev-Squad/website-status/pull/264/files

Reply:
Gotcha :+1: